packages/editor-ui/src/components/NodeSettings.vue

```diff
@@ -281,6 +343,42 @@ export default mixins(
 			},
 		},
 		methods: {
+			/**
+			 * Check if any of the node's credential types may be used to
+			 * make a request with the HTTP Request node v2.
+			 */
+			checkHttpRequestNodeSupport(credentialTypeNames: string[]) {
+				this.isSupportedByHttpRequestNode = credentialTypeNames.some(name => {
+					const credentialType = this.getCredentialTypeByName(name);
+
+					return (
+						credentialType.name.slice(0, -4).endsWith('OAuth') ||
+						credentialType.authenticate !== undefined
+					);
+				});
+			},
+			async prepareScopesNotice(activeCredentialType: string) {
+				if (
+					!this.isHttpRequestNodeV2(this.node) ||
+					!activeCredentialType || !activeCredentialType.endsWith('OAuth2Api')
+				) {
+					this.activeCredential.scopes = [];
+					return;
+				}
+
+				const credentialType = this.getCredentialTypeByName(activeCredentialType);
+
+				this.activeCredential.displayName = credentialType.displayName.replace('OAuth2 API', '').trim();
```

Comment:
not i18n friendly... how would this work once we add non oauth2 types?

Reply:
This already works with non-OAuth - replacing when nothing matches leaves the display name unaltered.

<img width="371" alt="image" src="https://user-images.githubusercontent.com/44588767/168257395-04b39666-0829-4d40-9553-2dcf10bb157b.png">

Reply:
Plain auth cred types do not have scopes.

Reply:
I see. can we address i18n issue then?

Reply:
It was done here: https://github.com/n8n-io/n8n/pull/3229/commits/9a7686f9e39aa071bc4b03a483526ca6022c308e

Or you mean something else?

Reply:
Likely this overseen commit is what was meant.